draft-ietf-httpbis-message-signatures.md

```diff
@@ -384,6 +387,82 @@ This example shows a canonicalized value for the parameters of a given signature
 
 Note that an HTTP message could contain multiple signatures, but only the signature parameters used for the current signature are included in this field.
 
+### Request-Response Signature Binding {#content-request-response}
+
+When a signed request message results in a signed response message, the `@request-response` identifier can be used to cryptographically link the request and the response to each other by including the identified request signature value in the response. This identifier has a single REQUIRED parameter:
+
+`key`
+: Identifies which signature from the response to sign.
+
+The value MUST contain the `sf-binary` representation of the signature value of the referenced request identified by the `key` parameter.
+
+For example, when serving this signed request:
+
+~~~ http-message
+POST /foo?param=value&pet=dog HTTP/1.1
+Host: example.com
+Date: Tue, 20 Apr 2021 02:07:55 GMT
+Content-Type: application/json
+Content-Length: 18
+Signature-Input: sig1=("host" "content-type");created=1618884475\
+  ;keyid="test-key-rsa-pss"
+Signature: sig1=:f0KeSHnWqp0SdVrZOpyx4fooj4Ez+sb25j+vSzcNe8S8P+wKIC\
+  j5Tu39yfSS2C5WzAyYy9zk+jNSRVrcVJ9/onaJp7dE2wcHFc80958491cf+YMK4h7\
+  SBkLMEtaUNHOQQo7e7b/NHOH20R/7RXFE98l51C4SSCtgREjxSOm3ceRGZSz0Ev+S\
+  PSlXxHn0OY7u4GHxSvPbr7ml/3H/+J4bJZu6juwWB7ym0AUfyuBFFFFmd/iu1WUtg\
+  je47Q4Q89G7SnADbSFZPhjWWyaNWOo2d3qjLGwx1dfA31hEB5NSxzc3UI7M5L9gp8\
+  F9oOKYvZsIIWQCUqtg+Gcs4u3wWa7Pqg==:
+
+{"hello": "world"}
+~~~
+
+This would result in the following unsigned response message:
+
+~~~ http-message
+HTTP/1.1 200 OK
+Date: Tue, 20 Apr 2021 02:07:56 GMT
+Content-Type: application/json
+Content-Length: 68
+
+{"from": "world", "busy": true, "your call is very important to us"}
```

Comment:
this here JSON isn't valid

Reply:
Do I look like a parser to you? :P

Thanks, I threw the signing test together quickly, and I'll fix that.

Reply:
Updated with more correcter JSON.